libs/core/src/components/pds-progress/test/pds-progress.e2e.ts

```diff
@@ -0,0 +1,43 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('pds-progress', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<pds-progress></pds-progress>');
+
+    const element = await page.find('pds-progress');
+    expect(element).toHaveClass('hydrated');
+  });
+
+  it('renders with a custom percent values', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<pds-progress percent="25"></pds-progress>');
+
+    const element = await page.find('pds-progress >>> progress');
+    expect(element.getAttribute('value')).toBe('25');
+  });
+
+  it('renders with animated progress', async () => {
```

Comment:
These tests seem to be covered in the unit tests since a value isn't changing, just being checked. One e2e test I could see is the percentage being one value, then being updated to another